src/cli/dev.test.ts

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { describe, it, expect, beforeEach, vi, Mock } from "vitest";
+import { Command } from "commander";
+import { promises as fs } from "fs";
+import prompts from "prompts";
+import devCommand from "./dev";
+import { EventEmitter } from "events";
+
+vi.mock("fs", async () => {
+  const actual = await vi.importActual<typeof import("fs")>("fs");
+  return {
+    ...actual,
+    promises: {
+      ...actual.promises,
+      writeFile: vi.fn(),
+    },
+  };
+});
+
+vi.mock("prompts", () => ({
+  default: vi.fn(),
+}));
+
+vi.mock("./build/loadModule", () => ({
+  loadModule: vi.fn().mockResolvedValue({
+    cfg: {
+      description: "test",
+      pepr: { uuid: "1234" },
+    },
+    path: "./test-module.js",
+  }),
+}));
+
+vi.mock("./build/buildModule", () => ({
+  buildModule: vi.fn().mockImplementation(async (_, cb) => {
+    await cb({ errors: [] });
+  }),
+}));
+
+vi.mock("../lib/assets/assets", () => ({
+  Assets: vi.fn().mockImplementation(() => ({
+    tls: {
+      pem: {
+        crt: "mock-cert",
+        key: "mock-key",
+      },
+    },
+    apiPath: "/mock/api",
+    capabilities: [{ name: "test-cap" }],
+    deploy: vi.fn().mockResolvedValue(undefined),
+  })),
+}));
+
+vi.mock("../lib/assets/deploy", () => ({
+  deployWebhook: vi.fn(),
+}));
+
+vi.mock("../lib/helpers", () => ({
+  validateCapabilityNames: vi.fn(),
+}));
+
+vi.mock("child_process", () => {
+  return {
+    fork: vi.fn(() => {
+      const mockChild = Object.assign(new EventEmitter(), {
+        kill: vi.fn(),
+        once: vi.fn((event, cb) => {
+          if (event === "exit") cb();
+        }),
+        on: vi.fn(),
+      });
+      return mockChild;
+    }),
+  };
+});
+
+vi.mock("kubernetes-fluent-client", () => ({
+  K8s: vi.fn(() => ({
+    Delete: vi.fn().mockResolvedValue(undefined),
+    InNamespace: vi.fn().mockReturnThis(),
+  })),
+  kind: {
+    MutatingWebhookConfiguration: "MutatingWebhookConfiguration",
+    ValidatingWebhookConfiguration: "ValidatingWebhookConfiguration",
+  },
+}));
+
+vi.mock("../lib/k8s", () => ({
+  Store: "Store",
+}));
+
+describe("dev command", () => {
+  let program: Command;
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+    program = new Command();
+    devCommand(program);
+  });
+
+  it("should run dev command successfully", async () => {
+    const writeFile = fs.writeFile as Mock;
+    await program.parseAsync(["node", "test", "dev", "--yes"]);
```

Comment:
```suggestion
    await program.parseAsync(["dev", "--yes"], {from: "user" });
```
Consistency with other tests we've written, though its functionally equivalent